"Deploy API Documentation" fails

See https://github.com/codeigniter4/CodeIgniter4/actions/workflows/deploy-apidocs.yml

2024-05-30T02:51:42.3525844Z ##[error]Process completed with exit code 1.

```
Run php tools/phpDocumentor run --ansi --verbose

In ArgvInput.php line 196:
                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  No arguments expected, got "run".                       
                                                          

Exception trace:
  at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Input/ArgvInput.php:196
 Symfony\Component\Console\Input\ArgvInput->parseArgument() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Input/ArgvInput.php:89
 Symfony\Component\Console\Input\ArgvInput->parseToken() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Input/ArgvInput.php:74
 Symfony\Component\Console\Input\ArgvInput->parse() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Input/Input.php:56
 Symfony\Component\Console\Input\Input->bind() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Command/Command.php:285
 Symfony\Component\Console\Command\Command->run() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Application.php:1096
 Symfony\Component\Console\Application->doRunCommand() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Application.php:324
 Symfony\Component\Console\Application->doRun() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/src/phpDocumentor/Console/Application.php:78
 phpDocumentor\Console\Application->doRun() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/vendor/symfony/console/Application.php:175
 Symfony\Component\Console\Application->run() at phar:///home/runner/.phive/phars/phpdocumentor-3.5.0.phar/bin/phpdoc:13
 require() at /home/runner/.phive/phars/phpdocumentor-3.5.0.phar:11

project:run [-t|--target [TARGET]] [--cache-folder [CACHE-FOLDER]] [-f|--filename [FILENAME]] [-d|--directory [DIRECTORY]] [--encoding [ENCODING]] [--extensions [EXTENSIONS]] [-i|--ignore [IGNORE]] [--ignore-tags [IGNORE-TAGS]] [--hidden] [--ignore-symlinks|--no-ignore-symlinks] [-m|--markers [MARKERS]] [--title [TITLE]] [--force] [--validate] [--visibility [VISIBILITY]] [--defaultpackagename [DEFAULTPACKAGENAME]] [--sourcecode|--no-sourcecode] [--template [TEMPLATE]] [--examples-dir [EXAMPLES-DIR]] [-s|--setting [SETTING]] [--list-settings] [--parseprivate]

Error: Process completed with exit code 1.
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/9296228744/job/25584421446

--- a/.github/workflows/deploy-apidocs.yml
+++ b/.github/workflows/deploy-apidocs.yml
@@ -61,7 +61,7 @@ jobs:
       - name: Build API in source repo
         working-directory: source
         run: |
-          php tools/phpDocumentor run --ansi --verbose
+          php tools/phpDocumentor --ansi --verbose
           cp -R ${GITHUB_WORKSPACE}/source/api/build/* ${GITHUB_WORKSPACE}/api/docs
 
       - name: Deploy to API repo

